### ISSUE TITLE
[Feature Request] Override + shortcut for Auto Dark Mode

### ISSUE THREAD

**Issue by pryoritymusic** [darkreader/darkreader#10570]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this feature request to avoid opening a duplicate.

### Is this feature request related to a problem?

No

### Feature Request Description

I love having Auto Dark Mode enabled so my screen goes dark at night.

However... it's a pain when dark mode sometimes breaks websites, and I have to go into the extension to disable it, rather than just running the shortcut Shift+Cmd+D. 

The ideal UX would basically be: 
- If Auto is enabled, toggle shortcut flips from dark-light or light-dark
- When next automatic transition is triggered, select dark or light based on auto settings

**Comment by bershanskiy** [comment]
1. Could you please report broken websites? We and the community might be able to fix them
2. Not sure what you are suggesting. For example, if automation is enabled, I'm currently in dark mode and transition to light mode will happen at T1 and transition to dark mode again at T2, and I click the shortcut, what happens at T1 and T2?

**Comment by pryoritymusic** [comment]
Hi, it's mainly sites that use off-color backgrounds - [one example](https://medium.com/hackernoon/managing-datadog-with-terraform-89abe0eb62f5)

If this is what it looks like with automation:
12am|------[T1]-------------------------[T2]--------------------|12am 
Dark-------                               ----------------------
Light.      -----------------------------

This is what I'd like to see:
12am|------[T1]-------------------------[T2]--------------------|12am---------[T1]
Dark-------          ---------[Tog]      --------[Tog]                            
Light.      --------[Tog].         -----          ------------------------------------->

Basically treat the auto-toggle times as transition states where at [T1] it turns on light mode regardless of the toggle, and at [T2] it turns on dark.

**Comment by romulasry** [comment]
Sub.

**Comment by domeniczz** [comment]
> 1. Could you please report broken websites? We and the community might be able to fix them
> 2. Not sure what you are suggesting. For example, if automation is enabled, I'm currently in dark mode and transition to light mode will happen at T1 and transition to dark mode again at T2, and I click the shortcut, what happens at T1 and T2?

If I encounter some broken site when I was in "auto" mode (in dark mode), I need extension shortcut, which can disables darkreader, works to temporarily allow me to view the broken site in light mode.

If I am in "on" mode, I can use the shortcut to turn darkreader into "off" mode conveniently, but the shortcut fails to work in "auto" mode.
